actions/nodejs/secure-attestations-download/action.yml

```diff
@@ -0,0 +1,23 @@
+name: "Secure attestations download"
```

Comment:
Creating a dedicated Action that matches the "nodejs" builder end-users will be familiar with. I think this also allows us to update a builder without breaking others.